js/types.ts

```diff
@@ -8,6 +8,7 @@ export interface ModuleInfo {
   outputCode: string | null;
 }
 
+// tslint:disable:max-line-length
```

Comment:
tslint is not being applied to `.d.ts` files at all, even ones we author internally...  So when I renamed this, tslint picked up some linting errors.